content/actions/hosting-your-own-runners/using-self-hosted-runners-in-a-workflow.md

```diff
@@ -70,9 +70,17 @@ These labels operate cumulatively, so a self-hosted runner’s labels must match
 
 When routing a job to a self-hosted runner, {% data variables.product.prodname_dotcom %} looks for a runner that matches the job's `runs-on` labels:
 
-1. {% data variables.product.prodname_dotcom %} first searches for a runner at the repository level, then at the organization level{% ifversion ghes or ghae %}, then at the enterprise level{% endif %}. 
+{% ifversion fpt or ghes > 3.2 or ghae %}
```

Comment:
this behavior is not on for GHAE M1, not sure whether that matters to the doc.

Reply:
🤔 Hmm, ok; I'll edit this for `-next`